src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+abstract class Figure implements AreaCalculator, Drawable {
```

Comment:
Don't use abstract classes to set behavior for classes. Consider whether `Figure` should be an abstract class or if an interface would be more appropriate here, depending on whether there are shared states or methods with default behavior.